docs/it/token_endpoint.rst

```diff
@@ -61,66 +61,60 @@ L’unico metodo di autenticazione all’endpoint token previsto è il private_k
+	 **iss**: Identificatore del RP registrato presso gli OP e che contraddistingue univocamente l'entità nella federazione nel formato Uniform Resource Locator (URL). Corrisponde al client_id usato nella richiesta di autenticazione 
 	 
 	 **sub**: uguale al parametro **iss** 
 	 
-	 **aud**: URL del Token Endpoint dell’OP
+	 **aud**: URL del Token Endpoint dell'OP
 	 
-	 **iat**: data/ora in cui è stato rilasciato il JWT in formato NumericDate, come indicato in RFC 7519 JSON Web Token (JWT). 
+	 **iat**: UNIX Timestamp con l'istante di generazione del JWT, codificato come NumericDate come indicato in :rfc:`7519`. 
 	 
-	 **exp**: data/ora di scadenza della request in formato NumericDate, come indicato in RFC 7519 – JSON Web Token (JWT). 
+	 **exp**: UNIX Timestamp con l'istante di scadenza del JWT, codificato come NumericDate come indicato in :rfc:`7519`
 	 
-	 **jti**: Identificatore univoco per questa richiesta di autenticazione, generato dal client casualmente con almeno 128bit di entropia.
-     -
+	 **jti**: Identificatore univoco uuid4 per questa richiesta di autenticazione, generato dal client casualmente con almeno 128bit di entropia.
      - |check-icon|
    * - **client_assertion_type**
-     -  
-     - Deve assumere il seguente valore: **urn:ietf:params:oauth:client-assertion-type:jwtbearer**
+     - Deve assumere il seguente valore: |br|
+       **urn:ietf:params:oauth:client-assertion-type:jwtbearer**
      - |check-icon|
    * - **code**
-     - Codice di autorizzazione restituito nell’Authentication response.
-     - 
+     - Codice di autorizzazione restituito nell'Authentication response.
      - Solo se **grant_type** è **authorization_code**
    * - **code_verifier**
      - Codice di verifica del code_challenge 
-     - 
      - Solo se **grant_type** è **authorization_code** 
    * - **grant_type**
      - Tipo di credenziale presentata dal Client per la richiesta corrente.
-     - Può assumere uno dei seguenti valori: 
+       Può assumere uno dei seguenti valori: 
 	 
-	 **authorization_code** 
-	 **refresh_token**
+	   **authorization_code** |br|
+	   **refresh_token**
      - |check-icon|
    * - **refresh_token**
-     -
      - 
      - Solo se **grant_type** è **refresh_code**   
  
  
 Response
 ++++++++
 
-Dopo avere ricevuto e validato la Token request dal client, il Token endpoint dell’OpenID Provider (OP) restituisce una response che include ID Token e Access Token e un eventuale Refresh Token, in formato JWT e firmati secondo le modalità definite dall’Agenzia per l’Italia Digitale.
+Dopo avere ricevuto e validato la Token request dal client, il Token endpoint dell'OpenID Provider (OP) restituisce una response che include ID Token e Access Token e un eventuale Refresh Token, in formato JWT e firmati secondo le modalità definite dall'Agenzia per l'Italia Digitale.
```

Comment:
Rimuovere " secondo le modalità definite dall'Agenzia per l'Italia Digitale." perché out of scope per CIE